### ISSUE TITLE
Bug: Post just saved can not view on frontpage because always add 7 hour

### ISSUE THREAD

**Issue by agussuhartono** [writesdown/app-cms#10]
Post just saved can not view on frontpage because always add 7 hour

this is caused by following code

$model->date = Yii::$app->formatter->asDatetime($model->date, 'php:Y-m-d H:i:s');

but on frontend the formatter not used

**Comment by 13nightevil** [comment]
I think to use `date('M d, Y H:i:s', strtotime($model->date))` instead of using `Yii::$app->formatter->asDatetime($model->date)` to show the datetime and `date('Y-m-d H:i:s', strtotime($model->date))` to save the data on the backend application. What do you think about that?

**Comment by agussuhartono** [comment]
It's good to use 
Yii::$app->formatter->asDatetime($model->date)  or 
date('M d, Y H:i:s', strtotime($model->date))

but using Yii::$app->formatter->asDatetime($model->date)  is more good, because can work globally

i think the problem is about consistency, both on backend and frontend :D

**Comment by 13nightevil** [comment]
Yes, I agree with you about working globally but when using `Yii::$app->formatter->asDatetime($model->date)` to save data, it will produce undesirable result because it depends on `Yii::$app->timeZone`, so I think to use `Yii::$app->formatter->asDatetime($model->date)` only to show datetime on frontend
